src/audio/host-zephyr.c

```diff
@@ -460,12 +452,7 @@ static int host_copy_normal(struct host_data *hd, struct comp_dev *dev)
 	if (!copy_bytes)
 		return 0;
 
-	struct dma_cb_data next = {
-		.channel = hd->chan,
-		.elem = { .size = copy_bytes },
-	};
-	notifier_event(hd->chan, NOTIFIER_ID_DMA_COPY,
```

Comment:
This removes each notifier_event for NOTIFIER_ID_DMA_COPY but keeps notifier_register (in host-zephyr.c) for NOTIFIER_ID_DMA_COPY to host_dma_cb?